File: Unidad 5 C#/Programa7U5.cs
using System;

namespace programa7u5
{
    internal class Programa7U5
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese el subtotal: ");
            double subtotal = Convert.ToDouble(Console.ReadLine());

            Console.Write("Ingrese el IVA: ");
            double tasaIVA = Convert.ToDouble(Console.ReadLine());

            double iva = CalcularIVA(subtotal, tasaIVA);
            double total = CalcularTotal(subtotal, iva);

            Console.WriteLine("Subtotal: " + subtotal);
            Console.WriteLine("IVA: " + iva);
            Console.WriteLine("Total: " + total);
        }

        static double CalcularIVA(double subtotal, double tasaIVA)
        {
            return subtotal * (tasaIVA / 100);
        }

        static double CalcularTotal(double subtotal, double iva)
        {
            return subtotal + iva;
        }
    }
}

File: Unidad 5 Java/Programa7U5.java
package com.mycompany.programa7u5;
import java.util.Scanner;

public class Programa7U5 {

    public static void main(String[] args) {
         Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese el subtotal: ");
        double subtotal = sc.nextDouble();

        System.out.print("Ingrese el IVA : ");
        double tasaIVA = sc.nextDouble();

        double iva = calcularIVA(subtotal, tasaIVA);
        double total = calcularTotal(subtotal, iva);

        System.out.println("Subtotal: " + subtotal);
        System.out.println("IVA: " + iva);
        System.out.println("Total: " + total);
    }

    static double calcularIVA(double subtotal, double tasaIVA) {
        return subtotal * (tasaIVA / 100);
    }

    static double calcularTotal(double subtotal, double iva) {
        return subtotal + iva;
    }
}
